plugins/modules/ldap_inc.py

```diff
@@ -0,0 +1,250 @@
+  - Atomically increments the value of an attribute and return its new value.
+notes:
+  - When implemented by the directory server, the module uses the ModifyIncrement extension
+    defined in L(RFC4525, https://www.rfc-editor.org/rfc/rfc4525.html) and the control PostRead. This extension and the control are
+    implemented in OpenLdap but not all directory servers implement them. In this case, the
+    module automatically uses a more classic method based on two phases, first the current
+    value is read then the modify operation remove the old value and add the new one in a
+    single request. If the value has changed by a concurrent call then the remove action will
+    fail. Then the sequence is retried 3 times before raising an error to the playbook. In an
+    heavy modification environment, the module does not guarante to be systematically successful.
+  - This only deals with integer attribute of an existing entry. To modify attributes
+    of an entry, see M(community.general.ldap_attrs) or to add or remove whole entries,
+    see M(community.general.ldap_entry).
+  - The default authentication settings will attempt to use a SASL EXTERNAL
+    bind over a UNIX domain socket. If you need to use a simple bind to access
+    your server, pass the credentials in O(bind_dn) and O(bind_pw).
+author:
+  - Philippe Duveau (@pduveau)
+requirements:
+  - python-ldap
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+options:
+  dn:
+    required: true
+    type: str
+    description:
+      - The DN entry containing the attribute to increment.
+  attribute:
+    required: true
+    type: str
+    description:
+      - The attribute to increment.
+  increment:
+    required: false
+    type: int
+    default: 1
+    description:
+      - The value of the increment to apply.
+  method:
+    required: false
+    type: str
+    default: auto
+    choices: [auto, rfc4525, legacy]
+    description:
+      - If V(auto), the module determines automatically the method to use.
+      - If V(rfc4525) or V(legacy) force to use the corresponding method.
+extends_documentation_fragment:
+  - community.general.ldap.documentation
+  - community.general.attributes
+
+'''
+
+
+EXAMPLES = r'''
+- name: Increments uidNumber 1 Number for example.com
+  community.general.ldap_inc:
+    dn: "cn=uidNext,ou=unix-management,dc=example,dc=com"
+    attribute: "uidNumber"
+    increment: "1"
+  register: ldap_uidNumber_sequence
+
+- name: Modifies the user to define its identification number (uidNumber) when incrementation is successful
+  community.general.ldap_attrs:
+    dn: "cn=john,ou=posix-users,dc=example,dc=com"
+    state: present
+    attributes:
+      - uidNumber: "{{ ldap_uidNumber_sequence.value }}"
+  when: ldap_uidNumber_sequence.incremented
+'''
+
+
+RETURN = r'''
+incremented:
+  description:
+    - It is set to V(true) if the attribute value has changed.
+  returned: success
+  type: bool
+  sample:
+    - true
```

Comment:
The return value is not a list, but a single value.
```suggestion
  sample: true
```

Reply:
Sorry for that

Reply:
No worries, that's what reviews are for. (I'm more surprised that the sanity tests didn't complain about that...)